src/Reflection/ReflectionClass.php

```diff
@@ -596,6 +597,8 @@ private function addEnumMethods(EnumNode $node, array $methods): array
     {
         $internalLocatedSource = new InternalLocatedSource('', $this->getName(), 'Core');
         $createMethod          = function (string $name, array $params, Node\Identifier|Node\NullableType $returnType) use ($internalLocatedSource): ReflectionMethod {
+            assert($name !== '');
```

Comment:
I think it's worth documenting the `@param` on the closure instead, if compatible

Reply:
I've tried but Psalm does not understand it.